common/src/main/java/dev/lavalink/youtube/clients/Web.java

```diff
@@ -207,17 +207,35 @@ protected JsonBrowser extractPlaylistVideoList(@NotNull JsonBrowser json) {
     protected String extractPlaylistContinuationToken(@NotNull JsonBrowser videoList) {
         // WEB continuations seem to be slightly inconsistent.
         JsonBrowser contents = videoList.get("contents");
-
         if (!contents.isNull()) {
             videoList = contents;
         }
-
         return videoList.values()
-            .stream()
-            .filter(item -> !item.get("continuationItemRenderer").isNull())
-            .findFirst()
-            .map(item -> item.get("continuationItemRenderer").get("continuationEndpoint").get("continuationCommand").get("token").text())
-            .orElse(null);
+                .stream()
+                .filter(item -> !item.get("continuationItemRenderer").isNull())
+                .findFirst()
+                .map(item -> {
+                    JsonBrowser continuationRenderer = item.get("continuationItemRenderer");
+                    JsonBrowser continuationEndpoint = continuationRenderer.get("continuationEndpoint");
+                    // first try if token is at : // continuationEndpoint.continuationCommand.token
+                    String token = continuationEndpoint.get("continuationCommand").get("token").text();
+                    if (token != null && !token.isEmpty()) {
+                        return token;
+                    }
+                    // second try if token is at : continuationEndpoint.commandExecutorCommand.commands[1].continuationCommand.token
+                    JsonBrowser commandExecutor = continuationEndpoint.get("commandExecutorCommand");
+                    if (!commandExecutor.isNull()) {
+                        JsonBrowser commands = commandExecutor.get("commands");
+                        if (!commands.isNull() && commands.isList() && commands.values().size() > 1) {
+                            return commands.values().get(1)
+                                    .get("continuationCommand")
+                                    .get("token")
+                                    .text();
+                        }
+                    }
+                    return null;
```

Comment:
Redundant null return. Just use the result from `return commands.values().get(1).get("continuationCommand").get("token").text();` which returns a nullable string.

Reply:
On second glance, `.values()` is redundant here. Use `commands.index(1).get("...")`